src/transport/tests/TestSecureSessionMgr.cpp

```diff
@@ -251,7 +251,7 @@ void SendEncryptedPacketTest(nlTestSuite * inSuite, void * inContext)
     payloadHeader.SetProtocolID(chip::Protocols::kProtocol_Echo);
 
     // Set the message type for this header.
-    payloadHeader.SetMessageType(chip::Protocols::kEchoMessageType_EchoRequest);
+    payloadHeader.SetMessageType(chip::Protocols::Echo::kEchoMessageType_EchoRequest);
```

Comment:
nit: some time later we may want to reduce the stuttering of **Echo**::k**Echo**MessageType_**Echo**Request